cfgov/jinja2/v1/_includes/molecules/contact-email.html

```diff
@@ -18,10 +16,10 @@
 {% macro render(emails) %}
 <div class="m-contact-email">
     {% set icon = 'cf-icon-email' %}
-    {% set title = 'Email' %}
+    {% set label = 'Email' %}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/6b0d6a8e6bc13dfa3ca7579d861109431ef24f57